examples/post_training_quantization/openvino/yolov8/main.py

```diff
@@ -0,0 +1,174 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import subprocess
+from tqdm import tqdm
+from pathlib import Path
+from typing import Dict
+import re
+
+import numpy as np
+import openvino.runtime as ov
+import nncf
+import torch
+
+from ultralytics import YOLO
+from ultralytics.yolo.utils.metrics import ConfusionMatrix
+from ultralytics.yolo.utils import DEFAULT_CONFIG
+from ultralytics.yolo.configs import get_config
+from ultralytics.yolo.utils import ops
+from ultralytics.yolo.data.utils import check_dataset_yaml
+
+
+def validate(model:ov.Model, data_loader:torch.utils.data.DataLoader, validator, num_samples:int = None, ):
```

Comment:
```suggestion
def validate(model:ov.Model, data_loader:torch.utils.data.DataLoader, validator, num_samples:int = None):
```
Please, also update type hints for input arguments and returns here and below.

Reply:
Of course, I can do that but I am not sure about the added value of that for the example and the user.